NuGet.Docs/Release-Notes/NuGet-3.0-Beta.md

```diff
@@ -0,0 +1,43 @@
+# NuGet 3.0 Beta Release Notes
```

Comment:
Please call out explicitly that users need to manually uninstall any previous installed NuGet extension first and then install the NuGet 3.0.0-Beta from Codeplex or extension gallery. We have the upgrade NuGet issue here - https://github.com/NuGet/Home/issues/129. Otherwise NuGet won't work properly.